src/Symfony/Component/Serializer/composer.json

```diff
@@ -16,12 +16,12 @@
         }
     ],
     "require": {
-        "php": ">=5.5.9"
+        "php": ">=5.5.9",
+        "symfony/property-access": "^3.1"
```

Comment:
not sure this should be moved as the serializer can be used without it with some normalizers.